composer.json

```diff
@@ -13,10 +13,10 @@
     ],
     "require": {
         "php": ">=7.1.3",
-        "symfony/framework-bundle": "^4.4|^5.0",
-        "symfony/dependency-injection": "^4.4|^5.0",
-        "symfony/routing": "^4.4|^5.0",
-        "symfony/http-foundation": "^4.4|^5.0",
+        "symfony/framework-bundle": "^4.4|^5.2",
+        "symfony/dependency-injection": "^4.4|^5.2",
+        "symfony/routing": "^4.4|^5.2",
+        "symfony/http-foundation": "^4.4|^5.2",
```

Comment:
Can you revert these? The bundle still works with 5.1 and 5.0. We will bump these at some point, but it doesn’t need to be here